values-schema.yaml

```diff
@@ -186,6 +186,9 @@ definitions:
             type: string
             description: Azure Application Client Secret
             x-secret: ''
+          secretName:
```

Comment:
I would rename to `secretNameRef` as the name already suggest that secret must exist

Reply:
I am sticking to the extern-dns value names